docs/platforms/apple/common/troubleshooting/index.mdx

```diff
@@ -115,6 +115,20 @@ SentrySDK.start { options in
     options.enableSwizzling = NO;
 }];
 ```
-## error: invalid version number in '-target arm64-apple-ios9999'
 
-If you see the error `error: invalid version number in '-target arm64-apple-ios9999'` when building your app, it's because you're using Xcode 15.4 or above with a Sentry SDK version that's lower than 8.26.0. To fix this, update your SDK to version 8.26.0 or above.
+## Xcode Compiler Error: Invalid Version
+
+If you see the error: `error: invalid version number in '-target arm64-apple-ios9999'` or `invalid version number in '-target arm64-apple-ios10.15-macabi'`
+when building your app, this is due to a bug on Xcode. To fix this issue, try to update the Cocoa SDK to version 8.26.0 or above or stay version 8.21.0.
+You can also try to switch to Sentry-Dynamic when integrating the Cocoa SDK via Swift Package Manager with the following steps:
+
+1. Open your .xcproject in Xcode.
+2. Select your target in Xcode under Targets.
+3. Navigate to Frameworks, Libraries, and Embedded Content.
+4. Click on the + sign to open a dialog for choosing frameworks and libraries to add.
+5. Select Sentry-Dynamic and click the Add button.
+6. Remove Sentry to ensure only Sentry-Dynamic appears under Frameworks, Libraries, and Embedded Content.
+7. Compile your project.
```

Comment:
```suggestion
1. Open your `.xcproject` in Xcode.
2. Select your target in Xcode under "Targets".
3. Navigate to "Frameworks, Libraries, and Embedded Content".
4. Click on the "+" sign to open a dialog for choosing frameworks and libraries to add.
5. Select "Sentry-Dynamic" and click "Add".
6. Remove "Sentry" to ensure only "Sentry-Dynamic" appears under "Frameworks, Libraries, and Embedded Content".
7. Compile your project.
```
Normally UI elements should be bolded but for some reason we put them in quotations in our docs, so adding some formatting here for consistency.